News: add 70 (2019-10-30)

harding opened the pull request.

harding commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -71,7 +71,24 @@ answers posted since our last update.*
+  Ugam Kamat explains how the proposed addition of schnorr signatures in
+  segwit v1 does not remove the need for ECDSA. ECDSA is required to spend
+  non-segwit as well as segwit v0 outputs and its removal would also require a
+  hard fork.
```
The SE answer does indeed say that removing ECDSA would require a hard fork, but it's wrong.  I'll argue over there, but I suggest changing it here.

jnewbery replied:
Agree. Harding has pointed out on SE that it might just be confusing wording, but I'd just remove entirely here. Perhaps change the sentence to "ECDSA will continue to be required to spend non-segwit as well as segwit v0 outputs."

jnewbery commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,129 @@
+## News
+
+- **LN simplified commitments:** in two separate threads, developers
+  from LND discussed their work on implementing simplified commitments,
```
'developers from Lightning Labs' or 'developers of LND' (Lightning Labs is the company, LND the product)

harding replied:
That's an important distinction, thank you for mentioning it.  I'll edit.

harding commented:
Pushed edit for @jnewbery feedback (thanks!), added a bullet for the Optech schnorr/taproot workshop, and added a short sentence and link to the PR for the LN simplified commitment previously described.

jnewbery commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,146 @@
+  notebooks, GitHub repositories, and more information produced for the
+  schnorr and taproot workshops held in San Francisco and New York
+  City last month.  These explain the fundamentals of both proposals,
+  guide students though actually using them, and then describe strategies
```
/students though/students through/

jnewbery commented:
One typo, otherwise ACK 19d5e9343e32540cfefd74ecdec8b643064b6641

harding commented:
Rebased on master to pull in the schnorr/taproot blog post, updated link to point to it, added C-Lightning release, and fixed typo identified by @jnewbery (thanks!).  All FIXMEs fixed now, so this should pass tests.

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+## Action items
+
+- **Upgrade to C-Lightning 0.7.3:** this latest [release][c-lightning
+  0.7.3] adds support for a PostgresSQL backend, makes it possible to
```
s/PostgresSQL/PostgreSQL/

jonatack replied:
possibly worth mentioning "in addition to SQLite3" (IIRC)

harding replied:
Thanks for catching that.  A note for anyone else reviewing: this is wrong in the C-Lightning release notes; I checked that the spelling @jonatack proposes is the same as being used by the [upstream PostgreSQL project](https://www.postgresql.org/).

jnewbery commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -17,10 +18,15 @@ the Bitcoin StackExchange.
+  and allows you keep your HD wallet seed encrypted when `lightningd`
+  isn't running---plus many other features and several bug fixes.
+
+- **Help Bitcoin Core release candidate:** experienced users are encouraged to
```
Should this be 'Help test Bitcoin Core release candidate'?

jonatack replied:
Agree. Perhaps "Help test the Bitcoin Core release candidate:"

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+
+- **Help Bitcoin Core release candidate:** experienced users are encouraged to
+  help test the latest release candidates for the upcoming version of
+  [Bitcoin Core][Bitcoin Core 0.19.0].
```
Perhaps provide a link to the [testing meta-issue](https://github.com/bitcoin/bitcoin/issues/17079) for test reports (or filing a separate issue in case of problems found).

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+
+## News
+
+- **LN simplified commitments:** in two separate threads, developers
```
possibly s/threads/mailing list|lightning-dev mailing list threads/

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+    LN protocol specification.
+
+- **Publication of videos and study material from schnorr/taproot workshop:**
+  Optech published a [blog post][taproot workshop] with links to videos, Jupyter
```
nit: sometimes Optech writes "jupyter" and other times "Jupyter"... possibly a candidate for the style guide

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+
+- [Why does hashing public keys not actually provide any quantum resistance?]({{bse}}91049)
+  Andrew Chow lists several considerations regarding public keys and quantum
+  resistance including: needing to reveal the public key during spending, the
```
nits: add a comma after "resistance", remove the colon, and s/needing/the need/

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -0,0 +1,152 @@
+  branch and will probably not be released until version 0.20, expected
+  about six months from now.  BIP70 was made optional in [version
+  0.18.0][core 0.18.0] and will be disabled by default in the upcoming
+  0.19.0, see [Newsletter #19][pr14451] for more information.
```
s/, see/. See/

moneyball commented:
ACK

harding commented:
Pushed (hopefully) final edits based on @jnewbery and @jonatack feedback (thanks).  I split the edits into two commits to make it easier to squash this down into separate commits for the newsletter text added by @bitschmidty and for the text added by myself.

jonatack commented on _posts/en/newsletters/2019-10-30-newsletter.md:
```diff
@@ -19,14 +19,15 @@ the Bitcoin StackExchange.
   help test the latest release candidates for the upcoming version of
-  [Bitcoin Core][Bitcoin Core 0.19.0].
+  [Bitcoin Core][Bitcoin Core 0.19.0] and [report][Bitcoin Core #17079]
+  any issues you find.
```
I don't know if you want to get into the details, but what is desired afaict is for people to report their RC testing in the meta-issue (to reassure the maintainers that testing is taking place and for which platforms) and to open a separate issue if a problem is encountered while testing.

harding replied:
That's confusing.  I'm going to drop mention of this issue.
